package.json

```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "lint": "eslint *.js bin lib tests",
     "test": "npm run-script lint && npm run-script utest",
-    "utest": "istanbul cover _mocha tests tests/ci tests/vcs"
+    "utest": "istanbul cover _mocha tests tests/ci tests/vcs",
+    "debug": "node $NODE_DEBUG_OPTION node_modules/.bin/_mocha tests tests/ci tests/vcs"
```

Comment:
Oh I've removed this now, but I did actually required the node_modules path, I think because I was  specifying the process to be run with `node $NODE_DEBUG_OPTION`. I needed it to be able to get interactive debugging working in webstorm